### DISCUSSION TITLE
Images not showing on deployed site, but work locally

### DISCUSSION THREAD

**Discussion by danaaronwhite** [vercel/community#7262] [Help]
### Summary

I am working on a Next.js 14 site and locally all of my images load fine, on the vercel url they work fine: https://launchculture-git-master-danaaronwhites-projects.vercel.app/ but live (https://www.launchculture.io) does not load the images. I haven't had this problem on other sites that are using the same exact system.

### Example

https://www.launchculture.io

### Steps to Reproduce

just visit the url and you'll notice no background images are loading so the page looks mostly white.

**Comment by versecafe** [comment]
@danaaronwhite Did you resolve this issue it appears to be working correctly?
<img width="1287" alt="image" src="https://github.com/vercel/community/assets/147033096/8f304b38-d1f5-455e-9526-244dfd7ee445">

[a comment by an automated account is not shown]
